BoldGrid/boldgrid-editor
# add tinymce_tables.js

resolves #445 

# Testing Instructions # 
1. Install PPB using the attached zip file in an environment where you have access to modify the wp-config.php file.
[post-and-page-builder-1.21.2-issue445.zip](https://github.com/BoldGrid/post-and-page-builder/files/9569448/post-and-page-builder-1.21.2-issue445.zip)

2. Add the following to your wp-config.php file `define( 'SCRIPT_DEBUG', true );` 
3. Edit or create a page / post, and ensure that the you do not see any errors regarding a missing `tinymce_tables.js` . Please note however, that with SCRIPT_DEBUG enabled, you are likely to see a lot of messages in the console that you do not normally see.
